src/Headers/EOS.h

```diff
@@ -429,7 +430,7 @@ class Radws : public EOS<ndim>
 
  public:
 
-  Radws(Parameters*, SimUnits *);
+  Radws(OpacityTable<ndim> *, Parameters*, SimUnits *);
```

Comment:
I think Radws can should construct the OpacityTable itself from the values in Parameters*.